drivers/ieee802154/ieee802154_rf2xx.c

```diff
@@ -133,6 +152,13 @@ static void rf2xx_trx_rx(struct device *dev)
 	 */
 	rf2xx_iface_frame_read(dev, rx_buf, RX2XX_FRAME_HEADER_SIZE);
 	pkt_len = rx_buf[RX2XX_FRAME_PHR_INDEX];
+	// NEED CHECK HERE IF EVERYTHING ABOVE ARE OK BEFORE DROP rf2xx_iface_frame_read
+	//pkt_len = ctx->rx_phr;
+	if (pkt_len != ctx->rx_phr)
+	{
+		LOG_INF("pkt_len != ctx->rx_pkt_len");
```

Comment:
The comment doesn't match the code, it seems that `rx_pkt_len` was renamed to `rx_phr`?